packages/package-bundler/README.md

```diff
@@ -8,165 +8,52 @@ ESM code is tree-shakeable.
 pnpm add -D @kaizen/package-bundler
 ```
 
-## build-shared-ui
+## Presets
 
-For shared UI packages. CSS modules and/or Tailwind will be included in the dist in a single stylesheet (`{PACKAGE_NAME}/dist/styles.css`) to be imported by the consumer.
+- [Default preset](src/presets/default/README.md)
+- [Shared UI preset](src/presets/shared-ui/README.md)
 
-_Note: If your package extends another shared UI package, you will need to list the other package as a peerDependency and have the end consumer import both stylesheets._
+## Package entrypoints
 
-### `package.json`
+Depending on the compatibility you wish to provide for your consumer, you may need to define your `package.json` differently.
+Read more about [the different types of package.json entrypoints here](https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions).
 
-Add the following to your `package.json`:
+### Node 12+
 
-```json
-"main": "dist/cjs/index.cjs", // CommonJS entrypoint
-"module": "dist/esm/index.mjs", // ES modules entrypoint
-"types": "dist/types/index.d.ts", // TypeScript types entrypoint
-"files": [
-  "dist" // Ensure dist dir is included in package
-],
-"sideEffects": [
-  "styles.css" // Ensure styles do not get tree-shaken
-],
-"scripts": {
-  "build": "pnpm package-bundler build-shared-ui",
-}
-```
-
-#### Dependencies
-
-```sh
-pnpm add -D postcss postcss-preset-env rollup tslib
-```
-
-`devDependencies`:
-
-- `postcss`
-- `postcss-preset-env`
-- `rollup`
-- `tslib` (peerDep of `rollup`)
-- If using Tailwind:
-  - `tailwindcss`
-
-### Required files
+If your consumers are all using Node 12+ (including for their TypeScript [`moduleResolution`](https://www.typescriptlang.org/tsconfig/#moduleResolution)), you should be able to use the [`exports`](https://nodejs.org/api/packages.html#exports) field if you have multiple entrypoints.
```

Comment:
I wonder if this needs to be more specific about which configuration of `moduleResolution` supports `exports`? 🤔